### DISCUSSION TITLE
Question - Is there a way to wildcard on files?

### DISCUSSION THREAD

**Discussion by scott-david-walker** [mikefarah/yq#1514] [Q&A]
If I have 10 files that all need the same value replaced, is it possible to do this? I can't find anything in the docs to say this is possible?

Thanks in advance

**Comment by mikefarah** [comment]
Nothing built in - but you can use `find`. I've added an example here: https://mikefarah.gitbook.io/yq/usage/tips-and-tricks#update-multiple-files
